config.py

```diff
@@ -0,0 +1,49 @@
+# Config Classes
+
+import os
+basedir = os.path.abspath(os.path.dirname(__file__))
+
+
+class Config(object):
+    SECRET_KEY = os.environ.get('SECRET_KEY') or 'Wo7GhuD2OWIv'
+    SQLALCHEMY_DATABASE_URI = os.environ.get('DATABASE_URL') or \
+        'sqlite:///' + os.path.join(basedir, 'app.db')
+    SQLALCHEMY_TRACK_MODIFICATIONS = False
+
+    ADMINS = ['your-email@example.com']
+
+    STATIC_ROOT = 'app/static/'
+    MEDIA_ROOT = 'media/'
+    MAX_CONTENT_LENGTH = 20 * 1024 * 1024
+    UPLOAD_PODCAST_FOLDER = 'episodes'
+
+    DROPZONE_DEFAULT_MESSAGE = 'Перетащите аудио файл в эту зону для загрузки'
```

Comment:
So there is a concept of detached string literals in the application, it is just not used consistently across the application.